index.md

```diff
@@ -450,44 +444,43 @@ escape, seguido por <code>:q!</code>(colon, olon, lower-case 'q',
           {% endif %}
           Instalador de Windowns
 	</a>
-	 y doble click en el archivo para correrlo.
+	 y haz doble click en el archivo para iniciar la instalación.
```

Comment:
I don't believe this installer still works. `nano` is now installed already with gitbash. This new section could be shortened (lines 436-447) with:
```html
          Nano debe de estar pre-instalado con la instalación de Git.
```